### PR TITLE
(MAINT) mention setup instructions on OpenBSD

### PR DESCRIPTION
**Pull request by jasperla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,15 @@ In Powershell:
 
 Note that OpenSSL isn't needed on Windows.
 
+### Setup on OpenBSD
```
Comment (kylog) [inline README.md]: Should this specify anything about which version of OpenBSD?

Comment (jasperla) [inline README.md]: There isn't a really strict requirement from the base system, rather the ports/packages have to be up to date. And base/packages out of sync is no go. So `OpenBSD 6.0` would be more accurate.

Comment (HAIL9000) [inline README.md]: Hey @jasperla, could you update these docs with the version so they're more accurate?

**Comment by jasperla** [comment]
Pardon the delay, it's properly annotated with a version now.

**Comment by whopper** [comment]
Thanks @jasperla!

**Pull request merged**